scalyr_agent/tests/scalyr_logging_test.py

```diff
@@ -21,30 +21,25 @@
 __author__ = "czerwin@scalyr.com"
 
 import os
-import re
 import tempfile
 from io import open
 
 import scalyr_agent.scalyr_logging as scalyr_logging
 
-from scalyr_agent.test_base import ScalyrTestCase
+from scalyr_agent.test_base import BaseScalyrLogCaptureTestCase
 
 
-class ScalyrLoggingTest(ScalyrTestCase):
+class ScalyrLoggingTest(BaseScalyrLogCaptureTestCase):
     def setUp(self):
         super(ScalyrLoggingTest, self).setUp()
-        self.__log_path = tempfile.mktemp(".log")
-        scalyr_logging.set_log_destination(
-            use_disk=True,
-            logs_directory=os.path.dirname(self.__log_path),
-            agent_log_file_path=self.__log_path,
-        )
+        self.__log_path = self.agent_log_path
         self.__logger = scalyr_logging.getLogger("scalyr_agent.agent_main")
         self.__logger.set_keep_last_record(False)
 
     def test_output_to_file(self):
         self.__logger.info("Hello world")
-        self.assertTrue(self.__log_contains("Hello world"))
+        self.assertLogFileContainsLineRegex(file_path=self.agent_log_path,
```

Comment:
Optional:  What do you think of having `file_path` default to `agent_log_path`?  So, if you do not specify `file_path`, we just check the agent log.  I can see the argument it's not as declarative about what's going on..but also, most of the times, we are checking the agent log so reduces typing.

Reply:
Yeah, I can make that change.

In fact that's what I did at first since that's what the tests have done originally, but then I changed my mind and went with the more "explicit" approach.

Reply:
I made a change with that argument being optional in ea484e0642d0bed5cf39d6a66c3fd79bfdb5e4dd.